HWLesson2.6/Devices/Model/ElectricDevice.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Enum;
+using Extension;
+
+namespace Model
+{
+    public class ElectricDevice : Device
+    {
+    public int Power { get; set; }
+
+    public ElectricDeviceType Type { get; set; }
+
+    public PowerSupply PowerSupply { get; set; }
+
+    public bool TurnON { get; set; }
+
+    public DateTime? DateTurnON  { get; set; }
+
+    public DateTime? DateTurnOff { get; set; }
```

Comment:
I suggest to use At in the end of the DateTime property like "TurnedOnAt" and "TurnedOffAt "